vaultclient/src/test/java/com/scality/vaultclient/services/AccountServicesClientTest.java

```diff
@@ -0,0 +1,268 @@
+
+        CreateAccountResponseDTO response = accountServicesClient.createAccount(createAccountRequestDTO).getAwsResponse();
+
+        assertEquals(DEFAULT_EMAIL_ADDR, response.getAccount().getData().getEmailAddress(), ERR_EMAIL_ADDR_INVALID);
+        assertEquals(DEFAULT_ACCOUNT_NAME, response.getAccount().getData().getName(), ERR_NAME_INVALID);
+        assertNotNull(response.getAccount().getData().getArn(), ERR_ARN_NULL);
+        assertNotNull(response.getAccount().getData().getCreateDate(), ERR_CREATE_DATE_NULL);
+        assertNotNull(response.getAccount().getData().getId(), ERR_ID_NULL);
+        assertNotNull(response.getAccount().getData().getCanonicalId(), ERR_CANONICAL_ID_NULL);
+    }
+
+    @Test
+    public void testCreateAccountWithCredentials() throws Exception {
+
+        CreateAccountResponseDTO response = accountServicesClient2.createAccount(createAccountRequestDTO).getAwsResponse();
+
+        assertEquals(DEFAULT_EMAIL_ADDR, response.getAccount().getData().getEmailAddress(), ERR_EMAIL_ADDR_INVALID);
+        assertEquals(DEFAULT_ACCOUNT_NAME, response.getAccount().getData().getName(), ERR_NAME_INVALID);
+        assertNotNull(response.getAccount().getData().getArn(), ERR_ARN_NULL);
+        assertNotNull(response.getAccount().getData().getCreateDate(), ERR_CREATE_DATE_NULL);
+        assertNotNull(response.getAccount().getData().getId(), ERR_ID_NULL);
+        assertNotNull(response.getAccount().getData().getCanonicalId(), ERR_CANONICAL_ID_NULL);
+    }
+
+    @Test
+    public void testCreateAccountWithQuota() throws Exception {
+
+        CreateAccountRequestDTO createAccountRequestDTO1 = CreateAccountRequestDTO.builder()
+                .emailAddress(DEFAULT_EMAIL_ADDR)
+                .name(DEFAULT_ACCOUNT_NAME)
+                .quotaMax(10)
+                .build();
+
+        CreateAccountResponseDTO response = accountServicesClient.createAccount(createAccountRequestDTO1).getAwsResponse();
+
+        assertEquals(DEFAULT_EMAIL_ADDR, response.getAccount().getData().getEmailAddress(), ERR_EMAIL_ADDR_INVALID);
+        assertEquals(DEFAULT_ACCOUNT_NAME, response.getAccount().getData().getName(), ERR_NAME_INVALID);
+        assertNotNull(response.getAccount().getData().getArn(), ERR_ARN_NULL);
+        assertNotNull(response.getAccount().getData().getCreateDate(), ERR_CREATE_DATE_NULL);
+        assertNotNull(response.getAccount().getData().getId(), ERR_ID_NULL);
+        assertNotNull(response.getAccount().getData().getCanonicalId(), ERR_CANONICAL_ID_NULL);
+        assertEquals(10, response.getAccount().getData().getQuotaMax(), "Invalid QuotaMax" );
+    }
+
+    @Test
+    public void testCreateAccountWithExtId() throws Exception {
+
+        CreateAccountRequestDTO createAccountRequestDTO2 = CreateAccountRequestDTO.builder()
+                .emailAddress(DEFAULT_EMAIL_ADDR)
+                .name(DEFAULT_ACCOUNT_NAME)
+                .externalAccountId("249349283982")
```

Comment:
Why are we hard coding account ID?

Reply:
@anurag4DSB 
This is a dummy account id and it can be any thing.
This is a unit test for CreateAccount when `externalAccountId` is provided in the request.